ams/server/src/main/java/com/netease/arctic/server/dashboard/utils/OptimizingUtil.java

```diff
@@ -63,34 +61,36 @@ private static FilesStatistics collectOptimizingFileInfo(MetricsSummary metricsS
     }
     return FilesStatistics.builder()
         .addFiles(metricsSummary.getEqualityDeleteSize(), metricsSummary.getEqDeleteFileCnt())
-        .addFiles(metricsSummary.getPositionalDeleteSize(), metricsSummary.getPosDeleteFileCnt())
+        .addFiles(metricsSummary.getPositionDeleteSize(), metricsSummary.getPosDeleteFileCnt())
         .addFiles(metricsSummary.getRewriteDataSize(), metricsSummary.getRewriteDataFileCnt())
         .build();
   }
 
-  public static long getFileSize(RewriteFilesOutput output) {
+  public static long getFileSize(ContentFile<?>[] contentFiles) {
     long size = 0;
-    if (output.getDataFiles() != null) {
-      for (DataFile dataFile : output.getDataFiles()) {
-        size += dataFile.fileSizeInBytes();
-      }
-    }
-    if (output.getDeleteFiles() != null) {
-      for (DeleteFile dataFile : output.getDeleteFiles()) {
-        size += dataFile.fileSizeInBytes();
+    if (contentFiles != null) {
+      for (ContentFile<?> contentFile : contentFiles) {
+        size += contentFile.fileSizeInBytes();
       }
     }
     return size;
   }
 
-  public static int getFileCount(RewriteFilesOutput output) {
+  public static int getFileCount(ContentFile<?>[] contentFiles) {
     int length = 0;
-    if (output.getDataFiles() != null) {
-      length += output.getDataFiles().length;
-    }
-    if (output.getDeleteFiles() != null) {
-      length += output.getDeleteFiles().length;
+    if (contentFiles != null) {
+      length += contentFiles.length;
```

Comment:
Return contents.length Directly

Reply:
The `contentFiles` is an array, and I believe this usage is fine 😊

Reply:
Done.